app/domain/queries/find_content.rb

```diff
@@ -38,12 +38,21 @@ def order_by
   def initialize(filter)
     @organisation_id = filter.fetch(:organisation_id)
     @document_type = filter.fetch(:document_type)
-    @search_term = filter[:search_term]
+    @search_term = parse_search_term(filter[:search_term]) if filter[:search_term]
     @page = filter[:page] || 1
     @page_size = filter[:page_size] || DEFAULT_PAGE_SIZE
     @date_range = filter.fetch(:date_range)
   end
 
+  def parse_search_term(search_term)
+    protocol = /http(s)?:\/\//
+    domain = /gov.uk/
```

Comment:
This might not work because GOV.UK URLs are always start with `www.gov.uk`. In content-tagger we've used `URI` to parse out the path instead:

https://github.com/alphagov/content-tagger/blob/1c4ab6fb7d99558250139a19db6702fadeff1bcf/app/forms/content_lookup_form.rb#L31-L36

Reply:
Thank you @tijmenb, I have added this commit (3b526f01e472a3697adbb54b75bf5bbf18421d5a) to ensure that we are considering `www.gov.uk` as a prefix that needs to be removed.

Reply:
Could you combine the into a single regex like `/(www.)?gov.uk/` ?

Reply:
I think you also need to escape `.` - `/(www\.)?gov\.uk/`